src/Counter/Counter.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+class Counter extends React.Component {
+  state = {
+    count: 0,
+  }
+
+  addOne = () => {
+    this.setState(state => ({
+      count: state.count + 1,
+    }));
+  };
+
+  add100 = () => {
+    this.setState(state => ({
+      count: state.count + 100,
+    }));
+  };
+
+  increase = () => {
+    this.addOne();
+    if (this.state.count % 5 === 0) {
```

Comment:
```suggestion
    if (this.state.count % 5 === 0 && this.state.count !== 0) {
```
Можно добавить еще такую проверку, чтобы ноль не считался числом делимым на 5